monkey/monkey_island/cc/resources/auth/registration.py

```diff
@@ -19,21 +21,13 @@ def get(self):
         return {"needs_registration": is_registration_needed}
 
     def post(self):
-        credentials = _get_user_credentials_from_request(request)
+        # TODO delete the old key here, before creating new one
```

Comment:
Is it worth it? I think it is. It will allow us to remove the extra step of `delete this mongo key` from credential reset procedure

Reply:
Makes sense, as long as it's appropriately encapsulated.